### PR TITLE
Allow configurable gunicorn concurrency

### PR DESCRIPTION
**Pull request by Cerberus98**

### CODE REVIEW

**Comment by Cerberus98** [comment]
Rebased on https://github.com/rackerlabs/quark/pull/411, please to merging that first

File: quark/gunicorn_server.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def load_config(self):
         self.cfg = gconfig.Config(self.usage, prog=self.prog)
         settings = {'bind': '%s:%s' % (cfg.CONF.bind_host, cfg.CONF.bind_port),
                     'workers': cfg.CONF.api_workers,
+                    'worker_connections': cfg.CONF.gunicorn.worker_connections,
```
Comment (jkoelker) [inline quark/gunicorn_server.py]: plz to be declare `worker_connections` in options for glorious `register_opts` to be inform.

Comment (Cerberus98) [inline quark/gunicorn_server.py]: done

**Comment by jkoelker** [comment]
LGTM

**Pull request merged**